docs/CONTAINER_INTERFACE.md

```diff
@@ -121,6 +121,16 @@ manager, please consider supporting the following interfaces.
    `container_ttys=pts/7 pts/8 pts/14` it will spawn three additional login
    gettys on ptys 7, 8, and 14.
 
+4. To allow applications to detect the OS version and other metadata of the host
+   running the container manager, please parse the host's `/etc/os-release` and
```

Comment:
Maybe "..., if this is considered desirable, please parse..." or something?

(The point I'm trying to make here is that we're not saying "you should expose this information to the container" so much as we are saying "*if* you *want* to expose this information to the container, here is how you do it")

Reply:
Thanks for the suggestion, fixed